main.cpp

```diff
@@ -52,106 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       car
+//  action 1:   the car accelerates
+car.accelerate();
+//  action 2:   the car brakes 
+car.brake();
+//  action 3:   the car takes a left turn
+car.takeLeftTurn();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        piano
+//  action 1:    the piano plays a note
+piano.playNote();
+//  action 2:    the piano plays a chord
+piano.playChord();
+//  action 3:    the piano plays a song
+piano.playSong();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        computer 
+//  action 1:    the computer turns on
+computer.turnOn();
+//  action 2:    the computer turns off
+computer.turnOff();
+//  action 3:    the computer runs a programm
+computer.runProgram();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        door
+//  action 1:    the door opens
+door.open();
+//  action 2:    the door closes
+door.close();
+//  action 3:    the door locks
+door.lock();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        lightBulb
+//  action 1:    the lightBulb turns on
+lightBulb.turnOn();
+//  action 2:    the lightBulb turns off
+lightBulb.turnOff();
+//  action 3:    the lightBulb illuminates
+lightBulb.illuminate();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        flower
+//  action 1:    the flower grows
+flower.grow();
+//  action 2:    the flower blooms
+flower.bloom();
+//  action 3:    the flower dies
+flower.die();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        bird
+//  action 1:    the bird flies
+bird.fly();
+//  action 2:    the bird chirps
+bird.chirp();
+//  action 3:    the bird eats
+bird.eat();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:        ball
+//  action 1:    the ball bounces
+ball.bounce();
+//  action 2:    the ball rolls
+ball.roll();
+//  action 3:    the ball flies
+ball.fly();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.